logagg_collector/collector.py

```diff
@@ -0,0 +1,926 @@
+import sys
+import time
+import glob
+import uuid
+import socket
+import datetime
+from operator import attrgetter
+import traceback
+from six import string_types as basestring
```

Comment:
@supriyopaul Please do reorder the imports, Refer the below thread of stackoverflow
Ref: https://stackoverflow.com/questions/20762662/whats-the-correct-way-to-sort-python-import-x-and-from-x-import-y-statement